prusti-viper/src/encoder/high/procedures/inference/semantics.rs

```diff
@@ -329,3 +354,43 @@ impl CollectPermissionChanges for vir_high::LeakAll {
         Ok(())
     }
 }
+
+impl CollectPermissionChanges for vir_high::NewLft {
+    fn collect(
+        &self,
+        _consumed_permissions: &mut Vec<Permission>,
+        _produced_permissions: &mut Vec<Permission>,
+    ) -> SpannedEncodingResult<()> {
+        Ok(())
+    }
+}
+
+impl CollectPermissionChanges for vir_high::EndLft {
+    fn collect(
+        &self,
+        _consumed_permissions: &mut Vec<Permission>,
+        _produced_permissions: &mut Vec<Permission>,
+    ) -> SpannedEncodingResult<()> {
+        Ok(())
+    }
+}
+
+impl CollectPermissionChanges for vir_high::GhostAssignment {
+    fn collect(
+        &self,
+        _consumed_permissions: &mut Vec<Permission>,
+        _produced_permissions: &mut Vec<Permission>,
+    ) -> SpannedEncodingResult<()> {
+        Ok(())
+    }
+}
+
+impl CollectPermissionChanges for vir_high::Borrow {
```

Comment:
Is this statement still used?

Reply:
yes, I believe it is needed for `struct Borrow` in `vir/defs/high/ast/statement.rs`.

Reply:
removed